docs/examples/Naive_Classification.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+Forward Algorithm
+=================
+An example of the forward algorithm using Stone Soup components.
+"""
+
+# %%
+# Modelling the state space
+# -------------------------
+# We model a target that can be one of three categories: Bicycle, Car or Bus.
+# In this simple instance, the target does not switch between these categories (ie. if it is a bus,
+# it stays a bus).
+# This is modelled in the state space by 3 state vector elements. The first component indicates
+# the probability that the target is a bicycle, the second that it is a car, and the third that it
+# is a bus.
+# For this model, it will remain a bus, therefore the 3rd state vector component will be
+# :math:`1` throughout, with the other components remaining :math:`0`.
+
+import numpy as np
+from stonesoup.types.state import State, StateVector
+
+np.random.seed(1991)
+
+gt = []
+nsteps=100
+
+for step in range(nsteps):
+    gt.append(State(StateVector([0, 0, 1])))  # bicycle, car, bus
+
+
+# %%
+# Modelling state transition
+# --------------------------
+# We require a model of the state transition in order to track the target's classification.
+# This will be a simple linear map, represented by a :math:`3\times 3` matrix with no noise added.
+# We might not know that the target does not switch classification, so the matrix should not simply
+# be the identity matrix.
+#
```

Comment:
This contradicts what's stated above ("ie. if it is a bus, it stays a bus"). 

It would be sensible to construct a model where the transition is the identity matrix, but noise can be added to account for the spontaneous transformation of a bus into a bike (presumably a very small amount in this instance. Of course the noise can't be merely additive, but you can use the `noise=True` flag to sample from a Categorical distribution with specified probabilities.

You could then use `noise=False` which would ensure that